usage.md

```diff
@@ -8,45 +8,43 @@ Installation will be required for capabilities such as [self-provisioning](doc/d
 
 ### Requirements
 
-The first three requirements derive from a placeholder install script. These will be addressed in #121.
+1. ESXi/vCenter - the target virtualization environment
```

Comment:
Do we need to mention the kernel OS? Did we move to photon kernel?
For configuration limitation, do you mean the configuration for ESXi/vCenter?